README.md

```diff
@@ -4,4 +4,129 @@ A tool for managing many combinations of real and fake NPM packages within your
 
 Extracted from [ember-auto-import](https://github.com/ef4/ember-auto-import). No docs here yet, but:
  - you can see how it's used in ember-auto-import/test-scenarios
- - it's all typescript so you can follow the types to learn what's possible
\ No newline at end of file
+ - it's all typescript so you can follow the types to learn what's possible
+
+## Getting started
+
+`scenario-tester` works especially well with a monorepo. We recommend that you create a new workspace package for your scenarios e.g. : 
+
+```
+mkdir test-packages/test-scenarios
+```
+
+Next you need to create some scenarios. The best way to create a scenario is from a Project, and the best way to create a project is to have a base project on disk. For example, in EmberJS you can create a new project with `ember new base-project` and it will create a new empty project for you. We can use this as a starting point for our other scenarios.
+
+```
+cd test-packages/test-scenarios
+ember new base-project
+```
+
+Next you can create a scenarios file that you can export a base project function: 
+
+
+```js
+// test-packages/tests-scenarios/scenarios.js
+import { Project } from 'scenario-tester';
+import { dirname } from 'path';
+
+// This is a way to allow for `require.resolve` to work in esm modules in node. 
+// If you are using CJS you can skip these two lines
+import { createRequire } from 'node:module';
+const require = createRequire(import.meta.url);
+
+export function baseApp() {
+  return Project.fromDir(dirname(require.resolve('./classic-app-template/package.json')), { linkDevDeps: true });
+}
+```
+
+While it's not quite that useful (yet) you can use this base app to build a specific test scenario. Let's do a simple test that boots your app.
+
+```js
+// test-packages/test-scenarios/boot-app-test.js
+
+import { baseApp } from './scenarios';
+import { Scenarios } from 'scenario-tester';
+import qunit from 'qunit';
+
+const { module: Qmodule, test } = qunit;
```

Comment:
Why do we alias these?
I don't think we need to in ESM?

Reply:
so in a real ESM setup you absolutely do because qunit isn't ESM 😞 In typescript we have been getting away with it for a while because it's so loosey-goosey with the whole thing 🤪 

I've been testing this in https://github.com/ef4/scenario-tester/pull/11 and https://github.com/ember-fastboot/ember-cli-fastboot/pull/919 and it doesn't work any other way 👍

Reply:
> so in a real ESM setup you absolutely do because qunit isn't ESM

can you expand on this? I've been using `module` with `qunit` in a real ESM environment (vite) no problem

Reply:
@NullVoxPopuli vite papers over a lot of incompatibilities for you so not sure if it's a good base to go off of

Reply:
oh sorry I thought you were talking about something else 🤔 I thought you were just telling me to do this: 

```
import { module as Qmodule, test } from 'qunit';
```

and that's what I was saying just doesn't work.

I didn't realise you were commenting on the fact that I was renaming module 🙃 The reason **I** did this is because I copy and pasted from... either ember-auto-import or embroider... I can't remember.

I never questioned it, but if it's not an issue I am happy to just use module directly 🤷 I assume Ed probably used it because there were a billion and a half uses of the word `module` in the codebase and it is easier to follow 😂